Golem.Package/Args.cs

```diff
@@ -5,7 +5,7 @@ public class BuildArgs
 {
     [Option('t', "target", Default = "package", Required = false, HelpText = "Directory where binaries will be generated relative to working dir")]
     public required string Target { get; set; }
-    [Option('y', "yagna-version", Default = "pre-rel-v0.16.0-ai-rc13", Required = false, HelpText = "Yagna version github tag")]
+    [Option('y', "yagna-version", Default = "pre-rel-v0.16.0-ai-rc15", Required = false, HelpText = "Yagna version github tag")]
```

Comment:
This RC includes a bugfix for remaining `ya-runtime-ai.exe` child process of `ya-provider.exe`.